darts/models/forecasting/sf_auto_arima.py

```diff
@@ -32,15 +32,11 @@ def __init__(
         It is probabilistic, whereas :class:`AutoARIMA` is not.
 
         We refer to the `statsforecast AutoARIMA documentation
-        <https://nixtla.github.io/statsforecast/models.html#arima-methods>`_
-        for the documentation of the arguments.
+        <https://nixtla.github.io/statsforecast/src/core/models.html#autoarima>`_
+        for the exhaustive documentation of the arguments.
 
         Parameters
         ----------
-        autoarima_args
-            Positional arguments for ``statsforecasts.models.AutoARIMA``.
-        autoarima_kwargs
-            Keyword arguments for ``statsforecasts.models.AutoARIMA``.
```

Comment:
I would revert the changes to this and the "note" section and do it similarly as we have it for TorchForecastingModels and the `**tfm_kwargs` arguments